src/wallet/wallet.cpp

```diff
@@ -4191,11 +4191,13 @@ util::Result<MigrationResult> MigrateLegacyToDescriptor(const std::string& walle
     std::vector<bilingual_str> warnings;
 
     // If the wallet is still loaded, unload it so that nothing else tries to use it while we're changing it
+    bool was_loaded = false;
```

Comment:
In commit "wallet: Reload the wallet if migration exited early" (d8002ed333e13e54726bcd922303cb1e42544b44)

I was confused by "As these checks do not change anything in the wallet, such unloaded wallets should be reloaded prior to exiting" in the commit description, since the wallets would need to be reloaded whether or not checks changed anything.

I think the commit message can just say "Such unloaded wallets should be reloaded prior to exiting" without the "As these checks" part

Reply:
Updated the commit message.